src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,60 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final int FIRST_ELEMENT_IN_LINE = 0;
+    private static final int SECOND_ELEMENT_IN_LINE = 1;
+    private int supply;
+    private int buy;
```

Comment:
this is not a very good practice to store state in service classes
so let's move these variables (supply, buy) to method

Reply:
I thought about it.
But I decided to take it out, because I use them in both (private void) methods.

Public only (public void getStatistic) to save file.

Do you insist ?